client/gutenberg/editor/main.jsx

```diff
@@ -53,13 +53,27 @@ class GutenbergEditor extends Component {
 	}
 }
 
+const addGutenbergDemoContent = post => {
+	const title = {
+		raw: 'Welcome to the Gutenberg Editor',
```

Comment:
We likely need to localize this I think.

Reply:
@Automattic/calypso-sdk @Automattic/i18n  any objections if we do the following?
```jsx
import { translate } from 'i18n-calypso';
//...
const addGutenbergDemoContent = post => {
	const title = {
		raw: translate( 'Welcome to the Gutenberg Editor' ),
		rendered: translate( 'Welcome to the Gutenberg Editor' ),
	};

	return {
		...post,
		title,
	};
};
```

Reply:
I've tested something like the following:
```
import { translate } from 'i18n-calypso';
//...
const demoTitleText = translate( 'Welcome to the Gutenberg Editor' );
const title = {
    raw: demoTitleText,
    rendered: demoTitleText,
};
```
Where a translation exists, it will render correctly. 

In this instance we have [translation](https://translate.wordpress.com/projects/wpcom/-all-translated/243089/), but you won't see translated output yet on `/gutenberg/post/{site}`.

I've added the source path `clients/gutenberg/editor/` to our [views](https://translate.wordpress.com/views/projects/wpcom/), so it should be picked up soon. I'll keep an 👁 on it. 👍

Reply:
If I understand correctly, this will always live in Calypso, so I don't see any downsides of using Calypso's i18n.

Reply:
Ah yes, I meant to double-check it was `translate`d at output, but no, it's not (which certainly makes sense). Will update.